### PR TITLE
DDP-5815: fix fields alignment, font-size and label

### PR DESCRIPTION
**Pull request by aglushko-broad**
before:
image

after:
image

### CODE REVIEW

File: ddp-workspace/projects/ddp-sdk/src/lib/components/activityForm/picklist/dropdownActivityPicklistQuestion.component.ts [hunk 1]
```diff
@@ -9,54 +9,60 @@ import { NGXTranslateService } from '../../../services/internationalization/ngxT
...
+                     [placeholder]="details.placeholder"
+                     [value]="details.text">
+          </mat-form-field>
+          <p *ngIf="!readonly" class="ddp-helper">
+              <span class="ddp-counter-color">
+                  {{ questionIdToCharactersLeft[details.stableId] }}
+              </span>{{ questionIdToCharacterLeftMsg[details.stableId] }}
+          </p>
+      </ng-container>
+    </mat-form-field>`,
```
Comment (MocanaAtBroad) [inline ddp-workspace/projects/ddp-sdk/src/lib/components/activityForm/picklist/dropdownActivityPicklistQuestion.component.ts:62]: @aglushko-broad : unfortunately git diff is not really helping to narrow down the change you made here. Could you briefly highlight/explain?

**Comment by alextanlan** [comment]
You improved the alignment.
However, `ddp-activity-answer`-s are still not on the same level.

For testing it  - I moved left block to the right as to overlap them (by adding `position: relative; left: 50px`)

- Before changes (on dev env):

before

- After changes (run your branch locally):

after

File: ddp-workspace/projects/ddp-sdk/src/lib/components/activityForm/picklist/dropdownActivityPicklistQuestion.component.ts (hunk 1 above)
Comment (aglushko-broad) [reply ddp-workspace/projects/ddp-sdk/src/lib/components/activityForm/picklist/dropdownActivityPicklistQuestion.component.ts:62]: sure! the changes I made:
- wrap with mat-form-field and assign matNativeControl to native select in order to apply material styles for alignment
- remove the first option from native select which was equal to the label and add mat-label (I had to add it out of native select ng-template in order to let internal mechanism which uses ng-content to handle it)

**Comment by aglushko-broad** [comment]
> You improved the alignment.
> However, `ddp-activity-answer`-s are still not on the same level.
> 
> For testing it - I moved left block to the right as to overlap them (by adding `position: relative; left: 50px`)
> 
> * Before changes (on dev env):
> 
> before
> 
> * After changes (run your branch locally):
> 
> after

nice catch! didn't see it on the small screen. Fixed

**Comment by alextanlan** [comment]
Perfect!

**Review by alextanlan — DISMISSED** [review]

**Comment by MocanaAtBroad** [comment]
OK. One super minor thing: the bottom of the "g" in the words "age" and "diagnosis" is getting cut off (it shows up in your screen capture too):
Screen Shot 2021-04-15 at 12 42 57 PM

**Comment by MocanaAtBroad** [comment]
Another thing I am noticing is that after introducing the Material field label, this particular question looks unbalanced (Age of diagnosis has Material label, but Cancer Type just placeholder). Looking if this something we can tweak from server configuration....

**Comment by MocanaAtBroad** [comment]
Let's back off/undo the change for Material label for the pull-down.

I think it might have wider repercussions/used in another apps and each of them would have to be verified that it looks good. Even in this case, just looking at code and configuration it is not totally clear what is the right thing to do now with the cancer type field while at the same time ensuring that it continuous to be right in all our apps.

Let's just make sure the alignment/original bug reported is right. We need that. At some point we can come back and revisit the Material label problem.

**Comment by aglushko-broad** [comment]
> Let's back off/undo the change for Material label for the pull-down.
> 
> I think it might have wider repercussions/used in another apps and each of them would have to be verified that it looks good. Even in this case, just looking at code and configuration it is not totally clear what is the right thing to do now with the cancer type field while at the same time ensuring that it continuous to be right in all our apps.
> 
> Let's just make sure the alignment/original bug reported is right. We need that. At some point we can come back and revisit the Material label problem.

thanks for noticing that label position for select doesn't match the position for autocomplete input! I fixed it quickly, you can take a look again. Though I see your concern that it affects all apps I'd say having the label is better than don't have a label at all, it's more noticeable in the selected state, then the user can't be sure what exactly s/he selected before.
This is how it looks now
image
But anyway it's not a problem to remove the label for now and return it how it was before. Could you please confirm one more time we want to do it, thanks!

**Comment by aglushko-broad** [comment]
> Let's back off/undo the change for Material label for the pull-down.
> 
> I think it might have wider repercussions/used in another apps and each of them would have to be verified that it looks good. Even in this case, just looking at code and configuration it is not totally clear what is the right thing to do now with the cancer type field while at the same time ensuring that it continuous to be right in all our apps.
> 
> Let's just make sure the alignment/original bug reported is right. We need that. At some point we can come back and revisit the Material label problem.

@MocanaAtBroad  it's all done! I updated the screenshot in the PR description

**Review by MocanaAtBroad — COMMENTED** [review]
Looks good! Let's merge.

**Review by alextanlan — APPROVED** [review]

**Pull request merged**